packages/paste-core/components/alert-dialog/src/AlertDialog.tsx

```diff
@@ -92,6 +95,35 @@ export interface AlertDialogProps extends HTMLPasteProps<"div"> {
   element?: BoxProps["element"];
 }
 
+interface NormalizeStylesArg {
+  opacity: SpringValue<number>;
+  transform: SpringValue<string>;
+}
+
+interface NormalizeStylesReturn {
+  opacity: number;
+  transform: string;
+}
+
+/**
+ * Normalize ReactSpring styles to be used in the AlertDialog.
+ *
+ * @param {NormalizeStylesArg} styles - ReactSpring styles
+ * @returns {NormalizeStylesReturn} - Normalized styles
+ */
+const normalizeStyles = (styles: NormalizeStylesArg): NormalizeStylesReturn => {
+  return {
+    ...styles,
+    opacity: styles.opacity.get(),
+    transform: styles.transform.get()
+  };
+}
```

Comment:
This function converts our SpringValue to the native generic value of the SpringValue. This normalizes the styles and allows us to avoid the any above.